src/main/java/com/artipie/vertx/VertxSliceServer.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.vertx;
+
+import com.artipie.asto.ByteArray;
+import com.artipie.http.RequestLine;
+import com.artipie.http.Slice;
+import io.reactivex.Flowable;
+import io.vertx.core.Handler;
+import io.vertx.reactivex.core.Vertx;
+import io.vertx.reactivex.core.buffer.Buffer;
+import io.vertx.reactivex.core.http.HttpServer;
+import io.vertx.reactivex.core.http.HttpServerRequest;
+import io.vertx.reactivex.core.http.HttpServerResponse;
+import java.util.Map;
+import org.reactivestreams.FlowAdapters;
+
+/**
+ * Vert.x Slice.
+ *
+ * @since 0.1
+ */
+public final class VertxSliceServer {
+
+    /**
+     * The Vert.x.
+     */
+    private final Vertx vertx;
+
+    /**
+     * The slice to be served.
+     */
+    private final Slice served;
+
+    /**
+     * The port to start server on.
+     */
+    private final Integer port;
```

Comment:
@Sammers21 why `Integer` instead of `int`?

Reply:
It does not matter